app/models/manageiq/providers/ansible_tower/shared/automation_manager/job.rb

```diff
@@ -134,9 +134,10 @@ def raw_status
     raise MiqException::MiqOrchestrationStatusError, err.to_s, err.backtrace
   end
 
-  def raw_stdout
+  def raw_stdout(format = nil)
```

Comment:
Should the `format = 'txt'` just live here?  What is the advantage to setting it below?